### DISCUSSION TITLE
0.2.4

### DISCUSSION THREAD

**Discussion by cawoodjm** [sfu-db/connector-x#231] [Q&A] [answered]
When are items in 0.2.4a2 expected to release in 0.2.4?

Joseph C.

**Comment by wangxiaoying** [comment, marked as answer]
Hi @cawoodjm , we plan to release 0.2.4 on Feb 28th.

**Reply by cawoodjm** [reply]
Did things get delayed?

**Reply by wangxiaoying** [reply]
Yeah, currently we are having some issues with refactoring our release pipeline. We will try to fix it or rollback to the old one if we cannot do it before this Friday.

**Reply by wangxiaoying** [reply]
Hi @cawoodjm , version 0.2.4 is released.
